### PR TITLE
Home button in debug menu

### PR DESCRIPTION
**Pull request by annelausf**
Home button on the debug menu does not return to home page. 

Fix: 
Change the way the href in the home button is generated. 
Made changes in all references to the home button in other pages.

### CODE REVIEW

File: pkg/sloop/webserver/webfiles/debug.html [hunk 1]
```diff
@@ -6,14 +6,27 @@
 -->
 <html>
 <head>
-    <script type="text/javascript">
-        document.write("<base href='/" + window.location.pathname.split('/')[1] + "/' />");
+    <!-- Load the base URL and sets Home button reference seperately -->
+    <script>
+        var homeref = "/" + window.location.pathname.split('/')[1];
```
Comment (venkatramreddykunta) [inline pkg/sloop/webserver/webfiles/debug.html:11]:
```suggestion
        var homeRef = "/" + window.location.pathname.split('/')[1]; 
```

File: pkg/sloop/webserver/webfiles/debug.html [hunk 2]
```diff
@@ -6,14 +6,27 @@
 -->
 <html>
 <head>
-    <script type="text/javascript">
-        document.write("<base href='/" + window.location.pathname.split('/')[1] + "/' />");
+    <!-- Load the base URL and sets Home button reference seperately -->
+    <script>
+        var homeref = "/" + window.location.pathname.split('/')[1]; 
+        function loadhomeref() {
```
Comment (venkatramreddykunta) [inline pkg/sloop/webserver/webfiles/debug.html:12]:
```suggestion
        function loadHomeRef() {
```

File: pkg/sloop/webserver/webfiles/debug.html [hunk 3]
```diff
@@ -6,14 +6,27 @@
...
+
     <title>Sloop Debug Menu</title>
     <link rel='shortcut icon' type='image/x-icon' href='webfiles/favicon.ico' />
 </head>
-<body>
-[ <a href="/">Home</a> ]<br/><br/>
+
+<body onload="loadhomeref()">
+[ <a id="homelink">Home</a> ]<br/><br/>
+
```
Comment (mengyaoyang11) [inline pkg/sloop/webserver/webfiles/debug.html:24]: Please remove those blank lines if added by mistake

File: pkg/sloop/webserver/webfiles/debug.html [hunk 4]
```diff
@@ -6,14 +6,27 @@
...
+
+
+<script>
```
Comment (mengyaoyang11) [inline pkg/sloop/webserver/webfiles/debug.html:27]: Any reason why we need the <script> tag here?

**Comment by mengyaoyang11** [comment]
Overall LGTM, would be great if it's possible to attach some before/after screenshots

File: pkg/sloop/webserver/webfiles/debug.html (hunk 1 above)
Comment (annelausf) [reply pkg/sloop/webserver/webfiles/debug.html:11]: Fixed thanks

**Comment by kritikapradhan** [comment]
Pls update the webfiles bindata.

File: pkg/sloop/webserver/webfiles/debug.html [hunk 5]
```diff
@@ -6,14 +6,19 @@
 -->
 <html>
 <head>
-    <script type="text/javascript">
-        document.write("<base href='/" + window.location.pathname.split('/')[1] + "/' />");
+    <!-- Load the base URL and sets Home button reference seperately -->
+    <script>
+        var homeRef = "/" + window.location.pathname.split('/')[1]; 
+        function loadHomeRef() {
```
Comment (kritikapradhan) [inline pkg/sloop/webserver/webfiles/debug.html:12]: Since this func and homeRef is common would it be better to define it once and reference ?

**Comment by kritikapradhan** [comment]
You would need to sign the salesforce-cla and tie ur commit to ur github.com username after, for the PR to be merged.

File: pkg/sloop/webserver/webfiles/debug.html (hunk 5 above)
Comment (venkatramreddykunta) [reply pkg/sloop/webserver/webfiles/debug.html:12]: +1
**Suggestion**: Move it to a separate js file and import it this all pages.

**Comment by venkatramreddykunta** [comment]
It's a good practice to maintain 1 commit for PR. 
**Suggestion**: squash all the commits.

**Pull request closed without merging**